cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,20 +1,28 @@
+import {AddressStep, LoginPage, MenuContentPage, PaymentStep, ProductListPage,
+  ShippingStep, ShoppingCartPage} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productListPage = new ProductListPage();
+const shoppingCartPage = new ShoppingCartPage();
+const loginPage = new LoginPage();
+const addressStep = new AddressStep();
+const shippingStep = new ShippingStep();
+const paymentStep = new PaymentStep();
+
 describe("Buy a t-shirt", () => {
   it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("#SubmitLogin").click();
-    cy.get("[name='processAddress']").click();
-    cy.get("[name='cgv']").click();
-    cy.get("[name='processCarrier']").click();
-    cy.get(".payment_module .bankwire").click();
-    cy.get(".cart_navigation span").click();
-    cy.get("#center_column > div > p > strong").should(
-        "have.text",
-        "Your order on My Store is complete.");
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productListPage.addTShirtToCart();
+    productListPage.proceedToCkeckout();
+    shoppingCartPage.proceedToCkeckoutLogin();
+    loginPage.loginToCheckOut();
+    loginPage.proceedToAddress();
+    addressStep.proceedToShipping();
+    shippingStep.acceptTermsOfService();
+    shippingStep.proceedToPayment();
+    paymentStep.payByBankWire();
+    paymentStep.confirmOrder();
+    paymentStep.checkSuccesfulyOrder();
```

Comment:
could you rename the method as validateConfirmationMessage or validateSucessfullMessage, and make it receive the message to validate as parameter

Reply:
Done